lib/open_food_network/feature_toggle.rb

```diff
@@ -1,13 +1,21 @@
 module OpenFoodNetwork
   class FeatureToggle
     def self.enabled?(feature)
-      !!features.with_indifferent_access[feature]
+      true?(value(feature))
     end
 
     private
 
-    def self.features
-      { connect_learn_homepage: false }
+    def self.value(feature)
```

Comment:
for extra clarity I'd name this method env_variable_value
and the variable feature could be name feature_name instead

Reply:
this would become easier to understand:
true?(env_variable_value(feature_name))